sp800-63b/sec5_authenticators.md

```diff
@@ -348,3 +348,15 @@ Biometrics are also used in some cases to prevent repudiation of registration an
 Authenticators that are directly connected to or embedded in endpoints MAY convey attestation information such as the provenance or health and integrity of the authenticator (and possibly the endpoint as well) to the verifier as part of the authentication protocol. If this attestation is signed, the verifier SHOULD validate its signature. This information MAY be used as part of a risk-based authentication decision.
 
 When federated authentication is being performed as described in [SP 800-63C](sp800-63c.html), the verifier SHOULD include any such attestation information in the assertion it provides to the relying party.
+
+#### <a name="verifimpers"></a>5.2.5 Verifier impersonation resistance
+
+Verifier impersonation attacks, sometimes referred to as "phishing attacks", refer to attempts by fraudulent verifiers and relying parties to fool an unwary claimant into authenticating to an impostor website. In previous editions of the Guideline, protocols that are resistant to verifier impersonation attacks were also referred to as "strongly man-in-the-middle resistant".
+
+Authentication protocols that are verifier impersonation resistant SHALL strongly and irreversibly bind the authenticator output to the hostname, domain, or public key of the certificate of the verifier to which it is sent. This makes the authenticator output useless to the attacker since it is not bound to the legitimate verifier.
```

Comment:
No, it gives a choice: hostname, domain name, or public key. FIDO U2F authenticators are also verifier impersonation resistant, but use the hostname rather than the public key AFAIK.